client/signup/steps/domains/index.jsx

```diff
@@ -762,6 +762,9 @@ class DomainsStep extends Component {
 		let isExternalBackUrl = false;
 
 		const previousStepBackUrl = this.getPreviousStepUrl();
+		const sitesBackLabelText = i18n.hasTranslation( 'Back to Sites' )
```

Comment:
Maybe it's a bit counter-intuitive, but `hasTranslation` always returns false for `en` users. So when I tested this I still saw the old `'Back to My Sites'` text.

You have to check for English locales as well unfortunately:
```suggestion
		const sitesBackLabelText = englishLocales.includes( this.props.locale )  || i18n.hasTranslation( 'Back to Sites' )
```

Where `englishLocales` is from `@automattic/i18n-utils`

Reply:
@p-jackson haha I had added the `en` check but forgot to push the changes, that's how I got the text to display correctly on the screenshot.

Reply:
strange all my screenshots and test instructions were not saved 😅